src/initialize.js

```diff
@@ -125,7 +125,10 @@ export const APP_INIT_ERROR = `${APP_TOPIC}.INIT_ERROR`;
  * as behavior may be undefined when managing history via multiple mechanisms/instances.
  *
  */
-export const history = createBrowserHistory();
+
+export const history = createBrowserHistory({
+  basename: process.env.BASENAME || '/',
```

Comment:
I initially did that (you can see the first commit of this PR) and it worked (and that is what I used to deploy using paths.) but in the tests that is failing. It says that getConfig() is undefined. Not sure what I have to do in order to fix these tests

Reply:
I wonder if there's a circular import... Ugh, that's probably it.  initialize.js imports `getConfig` from config.js, and config.js imports some constants from initialize.js.  Since `history` is evaluated as the file is parsed, `getConfig` isn't defined yet.  Something like that.

An easy way to see if this is the problem with the tests would be to temporarily stop importing `APP_CONFIG_INITIALIZED` in config.js and just dump its string value in `ensureConfig` instead - I bet the test starts passing then.  

If you're okay with a minor refactoring, I think the right move may be to move the constants (`APP_TOPIC`, `APP_CONFIG_INITIALIZED`, etc.) out of initialize.js into a constants.js file, which would keep config.js from needing to import initialize.js.  The index.js file would then re-export them from constants.js instead.  There may be other consumers that might need their imports updated as well.

Reply:
Hi @davidjoy, As you suggested, I created a constant.js file. Let me know if you think that it is Ok